### PR TITLE
FEAT: update to new version of Multi-View Reconstruction plugin

### PR DESCRIPTION
**Pull request by kozusznik**
in define_czi rule and "bsh" file: select_xml -> select; allow specify rotation and check stack size; and update other parameters

in export.bsh: select_xml -> select

in registration.bsh: select_xml -> select

in timelapse.bsh: select_xml -> select

in fusion.bsh: select_xml -> select; remove lossless flag; and update other parameters

fix registration.bsh: change variables downsample* to global scope
resave_hdf5_output rule: add resave_tile parameter

change paths for config yaml

### CODE REVIEW

File: spim_registration/timelapse/Snakefile [hunk 1]
```diff
@@ -543,6 +536,7 @@ rule resave_hdf5_output:
         -Dsetups_per_partition={setups_per_partition} \
         -Drun_only_job_number={job_number} \
         -Dfusion_switch={fusion_switch} \
+        -Dresave_tile={resave_tile} \
```
Comment (PetrBainar) [inline spim_registration/timelapse/Snakefile]: Please move this one a bit upwards so it is after the line where the `resave_timepoint` parameter is passed.

File: spim_registration/timelapse/cluster.json [hunk 2]
```diff
@@ -1,61 +1,62 @@
...
-        "lsf_extra" : "-n 12 -R \"span[hosts=1] rusage[mem=110000]\"",
-        "lsf_q" : "medium",
-        "slurm_extra" : "-n 1 -c 6 --mem-per-cpu=10000"
+        "lsf_extra" : "-n 6 -R \"span[hosts=1] rusage[mem=50000]\"",
+	"slurm_extra" : "-n 1 -c 6 --mem-per-cpu=10000"
     },
 
     "deconvolution" :
     {
+	"pbs_extra" : "-A DD-18-42",
```
Comment (PetrBainar) [inline spim_registration/timelapse/cluster.json]: Is this something we want to hold under version control?

File: spim_registration/timelapse/Snakefile [hunk 3]
```diff
@@ -41,24 +41,17 @@ rule define_xml_czi:
...
-        -Dangles={angles} \
-        -Dchannels={channels} \
-        -Dillumination={illumination} \
-        -Dmanual_calibration_czi={manual_calibration_czi} \
-        -Dczi_pixel_distance_x={czi_pixel_distance_x} \
-        -Dczi_pixel_distance_y={czi_pixel_distance_y} \
-        -Dczi_pixel_distance_z={czi_pixel_distance_z} \
-        -Dczi_pixel_unit={czi_pixel_unit} \
-        -Dfirst_xml_filename={first_xml_filename} \
-        -Drotation_around={rotation_around} \
```
Comment (PetrBainar) [inline spim_registration/timelapse/Snakefile:55]: A lot of parameters have been deleted here but they do appear in the commited `config.yaml`. Assuming that the `rotation_around` parameter, for example, is not used anymore. If this is the case, delete it also from the respective `config.yaml` section.

File: spim_registration/timelapse/define_czi.bsh [hunk 4]
```diff
@@ -14,180 +14,50 @@ print("Threads: "+Prefs.getThreads()); // prints thread setting in output
...
-	manual_calibration_string = "pixel_distance_x=" + czi_pixel_distance_x + " " +
-				    "pixel_distance_y=" + czi_pixel_distance_y + " " +
-				    "pixel_distance_z=" + czi_pixel_distance_z + " " +
-				    "pixel_unit=" + czi_pixel_unit + " ";
-	System.out.println( "Calibration set to manual" );
-	System.out.println( "modify_calibration:" + modify_calibration );
-	System.out.println( "manual_calibration_string:" + manual_calibration_string );
+apply_rotation_to_dataset = System.getProperty("apply_rotation_to_dataset");
+apply_rotation_to_dataset_str = "";
+if ( apply_rotation_to_dataset.equalsIgnoreCase( "Yes" ) ) {
```
Comment (PetrBainar) [inline spim_registration/timelapse/define_czi.bsh:24]: Maybe I am missing something but it looks like that information on the `No` or bad setting has disappeared. Is this intentional?

**Review by PetrBainar — CHANGES_REQUESTED** [review]

File: spim_registration/timelapse/define_czi.bsh (hunk 4 above)
Comment (kozusznik) [reply spim_registration/timelapse/define_czi.bsh:24]: Thats right - another value then "Yes" causes empty apply_rotation_to_dataset_str

Comment (PetrBainar) [reply spim_registration/timelapse/define_czi.bsh:24]: Sure, however this way the user won't be notified that there is an invalid setting, for example.

Comment (kozusznik) [reply spim_registration/timelapse/define_czi.bsh:24]: OK, that makes sense.

**Pull request merged**